public/claimlist.php

```diff
@@ -163,13 +165,14 @@
         }
         echo "<br style='clear:both'>";
 
-        echo "<div class='table-wrapper'><table class='table-highlight'><tbody>";
+        echo "<div class='table-wrapper'><table class='table-highlight text-xs'><tbody>";
 
-        echo "<tr class='do-not-highlight'>";
-        echo "<th></th>";
         // Sortable table headers
+        echo "<tr class='do-not-highlight'>";
```

Comment:
Suggest swapping the first two columns while you're in here to be consistent with the change in #1425.